source/numerics.tex

```diff
@@ -6752,12 +6752,6 @@
 matrix class nor a field class.
 However, it is a very useful building block for designing such classes.}
 
-\pnum
-An implementation is permitted to qualify any of the functions declared in
-\tcode{<valarray>}
-as
-\tcode{inline}.
-
```

Comment:
Yes, the statement is entirely redundant with [global.functions] p1

> It is unspecified whether any non-member functions in the C++ standard library are defined as inline (10.1.6).

and [member.functions] p1

> It is unspecified whether any member functions in the C++ standard library are defined as inline (10.1.6).